Source/WebKit/UIProcess/mac/PageClientImplMac.mm

```diff
@@ -293,6 +293,19 @@ - (BOOL)_hostsLayersInWindowServer;
 #if ENABLE(WRITING_TOOLS)
     m_impl->hideTextAnimationView();
 #endif
+
+#if ENABLE(SCREEN_TIME)
+    auto webView = this->webView();
+    NakedPtr<WebKit::WebPageProxy> pageProxy = [webView _page];
+    if (RetainPtr screenTimeWebpageController = [webView _screenTimeWebpageController]) {
+        // If there is a preexisting controller but Screen time is disabled, remove controller.
```

Comment:
Screen Time (capitalization?)

this is to make dynamically disabling the setting work for an existing view? I don't think it's strictly necessary that that works, but I guess it's nice that you considered it!